config/policies/storage_type_validation.yaml

```diff
@@ -0,0 +1,25 @@
+apiVersion: admissionregistration.k8s.io/v1
+kind: ValidatingAdmissionPolicy
+metadata:
+  name: "storage-type-validation"
+spec:
+  failurePolicy: Fail
+  matchConstraints:
+    resourceRules:
+    - apiGroups:   ["galaxy.ansible.com"]
+      apiVersions: ["*"]
+      operations:  ["CREATE", "UPDATE"]
+      resources:   ["galaxies"]
+  validations:
+  - expression: "has(object.spec.storage_type)"
+    message: "storage_type should not be empty"
+    reason: Invalid
+  - expression: "!(object.spec.storage_type in ['File', 'file'] && !has(object.spec.file_storage_storage_class))"
+    message: "please specify a rwx file_storage_storage_class"
+    reason: Invalid
```

Comment:
So this forces customers to be explicit.  The one downside is that it won't automatically use the default storage class on the cluster right?

Reply:
exactly